pages/index.tsx

```diff
@@ -113,7 +132,9 @@ export default function Home(props: { users: Array<any> }) {
                 shift={shift}
                 type={identifyShift(shift, userId)}
               />
-            ))}
+            ))
+            .filter((Shift)=>(
+              showShmirotOnly ? shmirot.includes(Shift.props.type.type): true))}
```

Comment:
That looks hacky. Is there another way to determine a shift is a shmira? Maybe exclude the ones marked as events and oncall?

Reply:
if u refer to this Shift.props.type.type, this is the type that came from the notion